### PR TITLE
Move perfy stuff into Perf Diagnostics report aggregation.

### PR DESCRIPTION
**Pull request by paulirish**
screenshot:
image

### CODE REVIEW

File: lighthouse-core/audits/dobetterweb/script-blocking-first-paint.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ class ScriptBlockingFirstPaint extends Audit {
     return {
       category: 'Performance',
       name: 'script-blocking-first-paint',
-      description: 'Avoids `<script>` in head that delay first paint',
+      description: 'Render-blocking `<script>`',
```
Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/script-blocking-first-paint.js]: should we be consistent? `<script>` -> scripts?

File: lighthouse-core/audits/byte-efficiency/uses-optimized-images.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ class UsesOptimizedImages extends Audit {
     return {
       category: 'Images',
       name: 'uses-optimized-images',
-      description: 'Avoids unoptimized images',
+      description: 'Unoptimized Images',
```
Comment (ebidel) [inline lighthouse-core/audits/byte-efficiency/uses-optimized-images.js]: Images -> image

sentence case everywhere.  PSI does that too.

screen shot 2017-02-14 at 7 01 26 pm

File: lighthouse-core/config/default.json [hunk 3]
```diff
@@ -372,20 +359,19 @@
       }
     }]
   }, {
-    "name": "Performance Metrics",
+    "name": "Performance diagnostics",
```
Comment (ebidel) [inline lighthouse-core/config/default.json]: would "Performance" be enough?

File: lighthouse-core/audits/byte-efficiency/unused-css-rules.js [hunk 4]
```diff
@@ -29,7 +29,7 @@ class UnusedCSSRules extends Audit {
     return {
       category: 'CSS',
       name: 'unused-css-rules',
-      description: 'Avoids loading unnecessary CSS',
+      description: 'Unused CSS Rules',
```
Comment (patrickhulce) [inline lighthouse-core/audits/byte-efficiency/unused-css-rules.js]: Haha, yeah might be worth having a dedicated report messaging discussion just to iron out what we want to standardize on :)

File: lighthouse-core/config/default.json [hunk 5]
```diff
@@ -372,20 +359,19 @@
...
       "audits": {
-        "critical-request-chains": {
-          "expectedValue": 0,
-          "weight": 1
-        },
-        "user-timings": {
-          "expectedValue": 0,
-          "weight": 1
-        }
+        "unused-css-rules": {},
```
Comment (patrickhulce) [inline lighthouse-core/config/default.json]: perf.json already looks like this, correct?

**Comment by paulirish** [comment]
PTAL. Updated.

Also updated perf.json as `--perf` is currently broken.

@ebidel If you're down, I'd like to file a ticket to address our green-checks-next-to-"unused css rules" bug in the report. I've already added it to [Lighthouse Report Requirements](https://docs.google.com/document/d/1A2JvCi6cQwHKB7MI_vTqTUF97xsimKQOb-KaOXFsWAQ/edit)

**Comment by ebidel** [comment]
IMO we should keep "Avoids..." until we've figured out something different. That language was hashed out over a  couple of PRs and had Kayce input. 

I'm on mobile, but there's a Pavel ticket already open that you can add to.

**Comment by paulirish** [comment]
Merged to fix #1775 and others. 

Will comment on #1613 and take the bug to address wording. Also #1750 by @sendilkumarn will help considerably with the icon/text combo.

**Pull request merged**